README.md

```diff
@@ -58,6 +58,41 @@ Note that supplying the service name for the destination service is optional;
 the tracing will default to a service name derived from the first section of the destination URL (e.g. 'service.example.com' => 'service').
 
 
+### Local tracing
+
+`ZipkinTracer::TraceClient` provides API to record local traces in your application.
```

Comment:
Maybe a one-liner about how a local trace is different from a normal trace would be helpful (for those new to the project)